test-ts/utils/parseTT.ts

```diff
@@ -1,3 +1,3 @@
 import { utils } from 'ethers'
 
-export const parseTT = (amount: number) => utils.bigNumberify(amount).mul(10 ** 8)
+export const parseTT = (amount: utils.BigNumberish) => utils.parseUnits(amount.toString(), 8)
```

Comment:
Isn't it essentially the same as `toTrustToken`? Maybe we should remove `toTrustToken` and use this one instead?

Reply:
And still maybe try to come up with more descriptive name

Reply:
Yeah, the problem is I didn't want to import from test and there's no a good place for utils function otherwise. Maybe moving to script/utils is a good idea

Reply:
We may have it in some main, non-test directory and import from there.